src/helpers/amoc_xmpp.erl

```diff
@@ -38,15 +38,15 @@ connect_or_exit(Spec) ->
             exit(connection_failed)
     end.
 
-%% @doc Picks a random server based on config var `xmpp_servers`
-%% This function expect a list of proplists defining the endpoint
+%% @doc Picks a random server based on config var `xmpp_servers'.
```

Comment:
```suggestion
%% @doc Picks a random server based on the config var `xmpp_servers'.
```
Also, why substituting `` ` `` by `'` ?

Reply:
Well, it's funny, but "\`xmpp_servers\`" will probably not compile as proper Edoc, as it will see it as an unfinished quotation. What we do in Markdown as "\`...\`" is done with "\`...'" in Edoc and I've noticed it's sometimes tricky for us to remember. Similarly, because Edoc does not support Markdown, there are no "*" lists in Edoc, as I've seen used in some places.